graphix/gflow.py

```diff
@@ -245,123 +140,71 @@ def find_flow(
 
     Parameters
     ----------
-    graph: :class:`networkx.Graph`
-        Graph (incl. input and output)
-    iset: set
-        set of node labels for input
-    oset: set
-        set of node labels for output
-    meas_planes: dict(int, Plane)
-        measurement planes for each qubits. meas_planes[i] is the measurement plane for qubit i.
-        Note that an underlying graph has a causal flow only if all measurement planes are Plane.XY.
-        If not specified, all measurement planes are interpreted as Plane.XY.
+    graph : `networkx.Graph`
+        The underlying graph.
+    iset : `collections.abc.Set`
+        Input nodes.
+    oset : `collections.abc.Set`
+        Output nodes.
 
```

Comment:
The documentation of the parameter `meas` is missing. Is that what you meant on line 154?